### PR TITLE
Update checks the min PHP version first

### PR DESCRIPTION
**Pull request by math-GH**
Ref. https://github.com/FreshRSS/FreshRSS/discussions/7141

Update checks the min PHP version first

grafik

grafik

### CODE REVIEW

File: scripts/update_to_1.25.0.php [hunk 1]
```diff
@@ -19,6 +20,11 @@ function apply_update() {
 		FRESHRSS_PATH . '/extensions',
 	);
 
+	// check PHP version requirement
+	if (version_compare(MIN_PHP_VERSION, phpversion(), '>')) {
```
Comment (math-GH) [inline scripts/update_to_1.25.0.php:24]: Do you prefer `phpversion()` or `PHP_VERSION`?

Comment (Alkarex) [reply scripts/update_to_1.25.0.php:24]: `PHP_VERSION` as this is what we use elsewhere, but I do not mind

**Review by Alkarex — APPROVED** [review]

**Comment by math-GH** [comment]
Ping @marienfressinaud for upload to the server

**Comment by marienfressinaud** [comment]
It's deployed!

**Pull request merged**